app/views/episode_media/media/_new.html.erb

```diff
@@ -1,93 +1,95 @@
-<div class="col-12 mb-4" data-controller="upload" data-upload-disable-outlet="form" data-morph="false">
-
-  <%# step 0: destroying media (will be nulled out if we actually replace it) %>
-  <%= form.hidden_field :id, data: {upload_target: "replace"} %>
-  <%= form.hidden_field :_destroy, value: true, data: {upload_target: "replace"} %>
-
-  <%# step 1: file field %>
-  <div class="form-floating" data-upload-target="picker">
-    <% cls = "form-control position-absolute opacity-0 z-1" %>
-    <% data = {action: "change->unsaved#change upload#upload", upload_target: "field"} %>
-    <% data[:unsaved_target] = "changed" if media.marked_for_destruction? %>
-
-    <%# NOTE: name is nil, so we don't submit the actual file to Rails %>
-    <% if episode.medium_video? %>
-      <%= form.file_field :media_file, class: cls, accept: "video/*", name: nil, data: data %>
-    <% else %>
-      <%= form.file_field :media_file, class: cls, accept: "audio/*", name: nil, data: data %>
-    <% end %>
-
-    <%# fake display field; real text area is opacity-0 on top of it %>
-    <% invalid_msg = upload_invalid_messages(media) %>
-    <% fake_cls = "form-control d-flex align-items-center prx_dragndrop" %>
-    <% fake_cls += " is-changed" if media.marked_for_destruction? %>
-    <% fake_cls += " is-invalid" if invalid_msg %>
-    <div class="<%= fake_cls %>">
-      <span class="material-icons text-primary">upload</span>
-      <div class="text-secondary mx-2 flex-grow-1 overflow-hidden text-truncate"><%= t(".hint") %></div>
-    </div>
+<% if policy(episode).update? %>
```

Comment:
To minimize the diff - I might move this into `media/_media.html.erb`.  And also use the `policy(media).create?` (assuming that works... can't remember if all the associations get set on there ahead of time).